templates/prescription/general_list.html.twig

```diff
@@ -0,0 +1,520 @@
+                <a href="{{ CONTROLLER }}?prescription&edit&id=0&pid{{  pid|js_url }}" class="btn-primary btn-sm  mr-3" id="addButton">{{ "Add"|xlt }}</a>
+                <a href="#" class="btn-secondary btn-sm quitButton">{{ "Quit"|xlt }}</a>
+            </div>
+        </div>
+    </div>
+</body>
+
+<script>
+
+$(function () {
+
+
+    prescriptionButtons.forEach(function(selector) {
+        $("#" + selector).on("click", function runConstraintsCheck(evt) {
+            for (var index in prescriptionButtonChecks[selector]) {
+                if (typeof prescriptionButtonChecks[selector][index] == 'function') {
+                    if (!prescriptionButtonChecks[selector][index](evt, this)) {
+                        evt.preventDefault();
+                        // we don't want other jquery events to occur if this fails
+                        evt.stopImmediatePropagation();
+                        return false;
+                    }
+                }
+            }
+            return true;
+        });
+
+    });
+    $(".check_list").on('click', function() {
+        let checkedNode = this;
+        prescriptionButtons.forEach(function (selector) {
+            changeLinkHref(selector, checkedNode.checked, checkedNode.value);
+
+        });
+        updateSendEmailIdList();
+    });
+    $(".editscript").on("click", function() {
+        ShowScript(this);
+    });
+    $(".onescript").on("mouseover", function() {
+        $(this).children().toggleClass("highlight");
+    });
+    $(".onescript").on("mouseout", function() {
+        $(this).children().toggleClass("highlight");
+    });
+    $(".btn-delete").on("click", function() {
+        deleteDrug(this.dataset['prescriptionId'])
+    });
+    $(".quitButton").on("click", function(evt) {
+        evt.preventDefault();
+        if (window.opener) {
+            dlgclose();
+        }
+    });
+    $("#sendEmailSectionCancel").on('click', function(evt) {
+        evt.preventDefault();
+        toggleEmailPrescriptionSection();
+    })
+    $("#sendPrescriptionForm").on('submit', function(evt) {
+        window.top.restoreSession();
+        let form = this;
+        // need to check to see if mailto submission is clicked
+        if (form.useDefaultMailClient.checked) {
+            evt.preventDefault();
+            $(form).find("input[type='submit']").attr("disabled", "disabled");
+            window.fetch("{{TOP_ACTION}}getDefaultMailClientText&ids=" + form.sendEmailPrescriptionIds.value)
+                .then(response => {
+                    if (response.ok) {
+                        return response.json();
+                    } else {
+                        throw new Error("Failed to receive valid text response back");
+                    }
+                })
+                .then(jsonContent => {
+                    let mailTo = "mailto:" + form.email_to.value
+                        + "?subject=" + jsonContent.subject
```

Comment:
would also encodeURIComponent the jsonContent.subject

Reply:
Fixed